buildpacks/nodejs-engine/src/bin/web_env.rs

```diff
@@ -0,0 +1,79 @@
+use libcnb::data::exec_d::ExecDProgramOutputKey;
```

Comment:
These attributes only apply to the current binary/library/integration test, so have to be specified in each one.

```suggestion
#![warn(clippy::pedantic)]

use libcnb::data::exec_d::ExecDProgramOutputKey;
```